src/app/kubernetes/components/pipeline-status/pipeline-status.component.scss

```diff
@@ -1,382 +1,554 @@
-    background-color: #d1d1d1
-}
-
-.pipeline-status-bar .inner-circle-fill {
-    background-color: #fff;
-    opacity: 0
-}
-
-.pipeline-status-bar.SUCCESS .clip1:before,.pipeline-status-bar.SUCCESS .clip2:before,.pipeline-status-bar.SUCCESS .inner-circle-fill,.pipeline-status-bar.SUCCESS .pipeline-line:before {
-    background-color: #3f9c35
-}
-
-.pipeline-status-bar.SUCCESS .pipeline-circle {
-    animation: fadeInSuccess 0s .7s linear forwards
-}
-
-.pipeline-status-bar.SUCCESS .pipeline-circle:after {
-    content: "\f00c"
-}
-
-.pipeline-status-bar.FAILED .clip1:before,.pipeline-status-bar.FAILED .clip2:before,.pipeline-status-bar.FAILED .inner-circle-fill,.pipeline-status-bar.FAILED .pipeline-line:before {
-    background-color: #c00
-}
-
-.pipeline-status-bar.FAILED .pipeline-circle {
-    animation: fadeInFailed 0s .7s linear forwards
-}
-
-.pipeline-status-bar.FAILED .pipeline-circle:after {
-    content: "\f00d"
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-line {
-    overflow: hidden
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-line:before {
-    animation: progress-rail 5s .5s linear infinite;
-    background-color: #0088ce;
-    transform: translateX(-100%);
-    width: 25%
-}
-
-.pipeline-status-bar.IN_PROGRESS .clip1:before,.pipeline-status-bar.IN_PROGRESS .clip2:before,.pipeline-status-bar.IN_PROGRESS .inner-circle-fill {
-    background-color: #0088ce
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-circle {
-    animation: fadeInProgress 0s .7s linear forwards
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-circle:after {
-    content: "\f021"
-}
-
-.pipeline-status-bar.NOT_EXECUTED .clip1:before,.pipeline-status-bar.NOT_EXECUTED .clip2:before,.pipeline-status-bar.NOT_EXECUTED .inner-circle-fill,.pipeline-status-bar.NOT_EXECUTED .pipeline-line:before {
-    background-color: #d1d1d1
-}
-
-.pipeline-status-bar.NOT_EXECUTED .pipeline-circle {
-    animation: fadeInNotExecuted 0s .7s linear forwards
-}
-
-.pipeline-status-bar.NOT_EXECUTED .pipeline-circle:after {
-    content: ""
-}
-
-.pipeline-status-bar.PAUSED_PENDING_INPUT .clip1:before,.pipeline-status-bar.PAUSED_PENDING_INPUT .clip2:before,.pipeline-status-bar.PAUSED_PENDING_INPUT .inner-circle-fill,.pipeline-status-bar.PAUSED_PENDING_INPUT .pipeline-line:before {
-    background-color: #f0ab00
-}
-
-.pipeline-status-bar.PAUSED_PENDING_INPUT .pipeline-circle:after {
-    content: "\f04c";
-    font-size: 10px
-}
-
-.pipeline-status-bar.ABORTED .clip1:before,.pipeline-status-bar.ABORTED .clip2:before,.pipeline-status-bar.ABORTED .inner-circle-fill,.pipeline-status-bar.ABORTED .pipeline-line:before {
-    background-color: #d1d1d1
-}
-
-.pipeline-status-bar.ABORTED .pipeline-circle {
-    animation: fadeInAborted 0s .7s linear forwards
-}
-
-.pipeline-status-bar.ABORTED .pipeline-circle:after {
-    content: "\f05e"
-}
+$build-pipeline-aborted-color: #d1d1d1;
```

Comment:
if this lives inside of OpenShift.io, it should match PatternFly, but I can see your point. I don't know when (or if) the OpenShift web console will switch over, but we can leave this as is until more information is found.